pandas/core/groupby/generic.py

```diff
@@ -443,18 +449,16 @@ def transform(self, func, *args, **kwargs):
         result.index = self._selected_obj.index
         return result
 
-    def _transform_fast(self, func, func_nm) -> Series:
+    def _transform_fast(self, func: Callable, func_nm: str) -> Series:
```

Comment:
If the arguments and/or return type of `Callable` can be provided it is very helpful; not sure how much incremental effort that is